api/models/graphite.go

```diff
@@ -79,22 +79,25 @@ func (gr GraphiteRender) Validate(ctx *macaron.Context, errs binding.Errors) bin
 }
 
 type GraphiteTags struct {
-	Filter string `json:"filter" form:"filter"`
+	tz.FromTo        // Used by GEM to decide what time range to return results for. Not used by upstream Graphite or Metrictank.
```

Comment:
I think many people outside Grafana don't know what `GEM` is, I'd just say "used by downstream projects" instead

Reply:
Makes sense, updated in https://github.com/grafana/metrictank/pull/2045/commits/d30600c8ca1cc7bfad945f736368837fcf4e3847